src/api/response.ts

```diff
@@ -12,14 +12,14 @@ export const jsonResponse = (res: Response, data: object) => {
 }
 
 export const errorResponse = (res: Response, error: ApiError, customMessage?: string, errorId?: string) => {
-  if (customMessage) error.errors = [{message: customMessage}]
+  if (customMessage) error.errors = [{ message: customMessage, code: '' }]
```

Comment:
tu trzeba cofnac ta zmiane nie? Inaczej res.status weźmie tego pustego stringa

Reply:
właśnie muszę tam dodać jakiś `code` bo typ `error.errors` tego wymaga

Reply:
no i `res.status` nie bierze stąd, tylko wyżej z `error.code`

Reply:
racja ale przeciez tam w typie jest `?` więc code nie jest wymagane https://github.com/softwaremill/sakiewka-crypto/pull/63/files#diff-a7b7d546b5dd6f539a5453eda2add2fbR3?

Reply:
@grubach

Reply:
tutaj ustalamy `ApiErrorDetails`, chodzi o to `code`
https://github.com/softwaremill/sakiewka-crypto/blob/bb2fa03e225b32995478f2c3add59858ca6e56a7/src/types/api.ts#L8

Reply:
@JakubDziworski